Fatras/include/ActsFatras/Kernel/InteractionList.hpp

```diff
@@ -78,17 +78,27 @@ using TupleFilter = typename TupleFilterImpl<predicate_t, tuple_t,
 
 /// Check if the given type is a point-like process.
 ///
-/// Only checks for the existence of the `generatePathLimits` method using the
-/// method described in https://stackoverflow.com/a/257382 .
+/// Only checks for the existence of the templated `generatePathLimits` method
+/// using the method described in https://stackoverflow.com/a/257382 .
 template <typename process_t>
 class IsPointLikeProcess {
-  using OneByte = char;
+  struct OneByte {
+    char x[1];
+  };
   struct TwoByte {
-    char x[2];
+    char y[2];
+  };
+  struct MockUniformRandomBitGenerator {
```

Comment:
I couldn't follow what the MockUniformRandomBitGenerator does or why it was introduced.

Reply:
This is part of #726 (which needs to be merged first) and not part of this PR. Since we do not require a specific generator, all process interfaces are templated on the [uniform random number generator type](https://en.cppreference.com/w/cpp/named_req/UniformRandomBitGenerator). Thus, in order to check if one of the member functions is defined, I need to instantiate the templated member function with a concrete generator type. I could have used any one of the standard generators in `#include <random>`. Since the type is never instantiated, and I do no actually use anything from `random`, I have a private mock type that acts exactly like an expected random number generator (as describe in the link above).